vllm/engine/ray_utils.py

```diff
@@ -30,12 +29,19 @@ def execute_method(self, method, *args, **kwargs):
             executor = getattr(self, method)
             return executor(*args, **kwargs)
 
+        def get_node_and_gpu_ids(self):
```

Comment:
nit: please annotate the return type.
```suggestion
        def get_node_and_gpu_ids(self) -> Tuple[int, List[int]]:
```